.github/workflows/docs.yaml

```diff
@@ -1,10 +1,8 @@
 name: Documentation Updates
 
 on:
-  push:
-    paths: ["Documentation/**"]
-  pull_request:
-    paths: ["Documentation/**"]
+  - push
+  - pull_request
```

Comment:
Nit: can be simplified to `on: [push,pull_request]`.

Reply:
@pchaigno that would mean exactly the same thing, do you prefer that style or you thought it means something else?

Reply:
I just preferred that style, but it isn't needed anymore :-)